sql/moz-fx-data-shared-prod/mozilla_org_derived/ga_sessions_v2/script.sql

```diff
@@ -0,0 +1,376 @@
+MERGE INTO
+  `moz-fx-data-shared-prod.mozilla_org_derived.ga_sessions_v2` T
+  USING (
+    WITH device_properties_at_session_start_event AS (
+      --get all session starts, from any date
+      SELECT
+        a.user_pseudo_id AS ga_client_id,
+        CAST(e.value.int_value AS string) AS ga_session_id,
+        (
+          SELECT
+            `value`
+          FROM
+            UNNEST(event_params)
+          WHERE
+            key = 'ga_session_number'
+          LIMIT
+            1
+        ).int_value AS ga_session_number,
+        geo.country AS country,
+        geo.region AS region,
+        geo.city AS city,
+        collected_traffic_source.manual_campaign_id AS campaign_id,
+        collected_traffic_source.manual_campaign_name AS campaign,
+        collected_traffic_source.manual_source AS source,
+        collected_traffic_source.manual_medium AS medium,
+        collected_traffic_source.manual_term AS term,
+        collected_traffic_source.manual_content AS content,
+        collected_traffic_source.gclid AS gclid,
+        device.category AS device_category,
+        device.mobile_model_name AS mobile_device_model,
+        device.mobile_marketing_name AS mobile_device_string,
+        device.operating_system AS os,
+        device.operating_system_version AS os_version,
+        device.language AS `language`,
+        device.web_info.browser AS browser,
+        device.web_info.browser_version AS browser_version,
+        PARSE_DATE('%Y%m%d', event_date) AS session_date,
+        ROW_NUMBER() OVER (
+          PARTITION BY
+            a.user_pseudo_id,
+            e.value.int_value
+          ORDER BY
+            a.event_timestamp ASC
+        ) AS rnk
```

Comment:
```suggestion
        ) AS row_num
```

This is just to more accurately represent that

Or something like that, rank is slightly different:
https://cloud.google.com/bigquery/docs/reference/standard-sql/numbering_functions#rank

Reply:
Fixed, thanks!